### DISCUSSION TITLE
AlDente no longer launches at log in

### DISCUSSION THREAD

**Discussion by fadic4** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#429] [Q&A]
Hello, 

I have AlDente version 1.08 installed on my MacBook Air M1 with macOS Monterey. 
I have the launch at log in turned on but what happens is that as soon as I log in, the AlDente launches then closes. 

is this an issue because of macOS Monterey and might be fixed later?

I already deleted and reinstalled AlDente but the issue still occurs.

**Comment by MatthiasKerbl** [comment]
> Hello,
> 
> I have AlDente version 1.08 installed on my MacBook Air M1 with macOS Monterey. I have the launch at log in turned on but what happens is that as soon as I log in, the AlDente launches then closes.
> 
> is this an issue because of macOS Monterey and might be fixed later?
> 
> I already deleted and reinstalled AlDente but the issue still occurs.

Hi @fadic4,

Thank you for reporting this issue. We are currently looking into this.

Cheers,
Matthias

**Comment by MatthiasKerbl** [comment]
Hi @fadic4,

We have just released version 1.09 of AlDente which has a fix for this issue.
https://github.com/davidwernhart/AlDente/releases

Cheers,
Matthias
